terraform/events.tf

```diff
@@ -0,0 +1,70 @@
+resource "aws_cloudwatch_event_rule" "scope_change" {
+  name        = "capture-tag-scope-hash-changes"
```

Comment:
Should we include site_name and/or component_name in it as well to avoid conflicts? As we do with `"${var.site}-${local.component_name}"` when naming the lambda function

Reply:
Good point, it needs to mirror https://github.com/labd/mach-component-aws-commercetools-token-refresher/blob/master/terraform/main.tf#L23 as we make it also unique per site and component else same component can't be deployed for another site in same aws account.

However this would trigger multiple rotations so have to limit to same site and component name, we should be able todo this with event bridge filter and a extra tag on the secret.

Reply:
@tleguijt please review this still needs to be tested on a mach deploy.

https://github.com/labd/mach-component-aws-commercetools-token-refresher/pull/4/commits/1e064b62c8aef57ea7c7a7bf552ead53015e28f4